src/Symfony/Component/HttpKernel/Controller/ArgumentResolverInterface.php

```diff
@@ -17,7 +17,7 @@
  * An ArgumentResolverInterface instance knows how to determine the
  * arguments for a specific action.
  *
- * @author Fabien Potencier <fabien@symfony.com>
+ * @author Iltar van der Berg <kjarli@gmail.com>
```

Comment:
I don't care too much, but this should be reverted. The code is exactly the same as the one from the original interface.